### DISCUSSION TITLE
Run test project/open editor in CI

### DISCUSSION THREAD

**Discussion by qarmin** [GodotECS/godex#252] [Ideas & Proposals]
This would allow to catch bugs like #218 without needing to manually test each commit.

In case if there is no test project, even simple opening and closing editor should help

**Comment by AndreaCatania** [comment]
_Moved this to discussion, to better keep track of the proposals._
This is a good idea, what's the best way to trigger that? Is it done in an unit test?

**Reply by qarmin** [reply]
In Godot, editor and test project are executed after unit tests(Godot 4.0 is able to run in CI only with SwiftShader, Lavapipe doesn't work for now), so I think that this is the best way to handle it
https://github.com/godotengine/godot/blob/9cb4fbe9d583e3f797e6b653fc9124515a7dd8f7/.github/workflows/linux_builds.yml#L159-L185

**Reply by AndreaCatania** [reply]
Ok, that seems pretty easy to add. However, I'll wait to add it until the linked bug is fixed, otherwise we will run into troubles.
